src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -474,12 +478,15 @@ async def determine_setup_kwargs(
 async def generate_chroot(request: SetupPyChrootRequest) -> SetupPyChroot:
     exported_target = request.exported_target
     exported_addr = exported_target.target.address
-
```

Comment:
Generally, it's best to avoid whitespace changes like this unless you have a particular reason for it. Tip when preparing a PR: I like to scan the change before hitting submit to check for stray changes I accidentally made like this, then I fix it up and do an interactive Git rebase and use the `fixup` command.

Reply:
Yeah I like GitHub's visual diff a lot for it.

Indeed, it is fairly tedious to do clean up things like whitespace. Although, it's valuable to do:
* Helps the reviewers because they have a smaller diff.
* Avoids polluting `git blame` / git history
* Whitespace is significant, and usually it was there before for better readability. We don't want to regress with that.

I also like using PyCharm's visual Git interface so that I can preview every change before committing. I usually fix things like this before I commit. There are other nice GUIs like Git Kraken too (altho, not required to use a GUI of course).